src/corefx/System.Globalization.Native/calendarData.cpp

```diff
@@ -122,24 +123,27 @@ const char* GetCalendarName(CalendarId calendarId)
 		case JAPANESELUNISOLAR:
 		case TAIWANLUNISOLAR:
 			return "chinese";
```

Comment:
Note that Japanese Lunisolar is different than Japanese calendar. I don't think it is right in our code to return Chinese calendar for Japanese calendar. 
By the way, the framework doesn't support Chinese calendar but it support Taiwanese calendar. I believe there is some differences between these calendars (I didn't look more though).